services/consensuscontext/test/harness.go

```diff
@@ -53,6 +54,7 @@ func (h *harness) requestTransactionsBlock(ctx context.Context) (*protocol.Trans
 		MaxNumberOfTransactions: 0,
 		PrevBlockHash:           hash.CalcSha256([]byte{1}),
 		PrevBlockTimestamp:      primitives.TimestampNano(time.Now().UnixNano() - 100),
+		PrevBlockReferenceTime:  primitives.TimestampSeconds(time.Now().Add(-10*time.Minute).Unix()),
```

Comment:
add a comment `// Genesis reference time`? just to stress that this function does not return any tx block - it only returns a genesis transactions block. Or, just rename the function to `requestGenesisTransactionsBlock`

Reply:
this is harness it doesn't return genesis tx it returns whatever was set in the mock.